packages/nakama-js/socket.ts

```diff
@@ -17,7 +17,9 @@
 import {ApiNotification, ApiRpc} from "./api.gen";
 import {Session} from "./session";
 import {Notification} from "./client";
-import {b64EncodeUnicode, b64DecodeUnicode} from "./utils";
+import {WebSocketAdapter, WebSocketAdapterText} from "./web_socket_adapter"
+import {b64DecodeUnicode, b64EncodeUnicode} from "./utils";
```

Comment:
We pull in the `"Base64"` library but also have a utils package which seems to work around non-UTF8 charsets to base64 decode the content? We should consolidate down to either one or the other.

Reply:
@novabyte the utils wrap `Base64`.